packages/page-staking/src/Performance/Performance.tsx

```diff
@@ -56,180 +49,67 @@ function Performance ({ className = '', favorites, sessionEra, toggleFavorite }:
   }, [eraExposure]
   );
 
-  useEffect(() => {
-    const sessionPeriod = Number(api.consts.elections.sessionPeriod.toString());
-    const firstBlockInSession = sessionEra.session * sessionPeriod;
-
-    api.rpc.chain
-      .getBlockHash(firstBlockInSession)
-      .then((result): void => {
-        setFirstBlockInSessionHash(result);
-      })
-      .catch(console.error);
-  },
-  [api, sessionEra]
-  );
+  const eraValidatorPerformances: EraValidatorPerformance[] = useMemo(() => {
```

Comment:
nit: since function inside memo is quite complex extract it out and just call it here

Reply:
Will do